app/graphql/types/organization_type.rb

```diff
@@ -27,6 +27,9 @@ class OrganizationType < Types::BaseObject
 
     field :timezone, Types::TimezoneEnum, null: true
 
+    field :document_numbering, Types::Organizations::DocumentNumberingEnum, null: false
+    field :document_number_prefix, String, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `document_number_prefix` should appear before `document_numbering`.